### DISCUSSION TITLE
Can outline editor support superscript / subscript?

### DISCUSSION THREAD

**Discussion by codingfunwoody** [outline/outline#4141] [General]
I think It's common feature, but outline doesn't support that

**Comment by Limezy** [comment]
I currently use the Unicode subscript or superscript but it's not very practical (only works for footnotes, basically)

**Reply by codingfunwoody** [reply]
I just know i can use unicode for the subscript or superscript, i hope it can be supported on the editor for convenience, anyway, this workaround can help me, thanks

**Comment by UKFr-DIZ** [comment]
You can also use inline Latex in oder to show it. 

`$$test_{subscript}$$`  
`$$test2^{superscript}$$ `

**Comment by mary-loulou** [comment]
Feature request: superscript / subscript:
- https://github.com/outline/outline/discussions/4141

Feature request: footnotes:
- https://github.com/outline/outline/discussions/2582
